docs/communication/props/props.md

```diff
@@ -1,3 +1,7 @@
+### Communication
+
+We're all set, now we know how to create our custom components and this will help us with code reuse and composition. One thing that is missing is how we can communicate between components. Vue.js provide us two simple but really powerful ways to pass data between our components and this will be pretty familiar if you are used to use HTML attributes.
```

Comment:
Change the word "code reuse" for "code reusability". Try to use more commas and points in this paragraph,  it's very extensive, while reading you tend to loose the sense of the sentence.